src/ConstructiveSolidGeometry/PointsAndVectors/Points.jl

```diff
@@ -20,7 +20,9 @@ struct CartesianPoint{T} <: AbstractCoordinatePoint{T, Cartesian}
     z::T
 end
 
-scale(pt::CartesianPoint{T}, s::SVector{3}) where {T} = CartesianPoint{T}(pt.x * s[1], pt.y * s[2], pt.z * s[3])
+zero(PT::AbstractCoordinatePoint{T}) where {T} = PT(zero(T),zero(T),zero(T))
```

Comment:
Is this right?
I would have expected something like
`zero(::PT) where {T, PT <: AbstractCoordinatePoint{T}} = PT(zero(T),zero(T),zero(T))` or 
`zero(PT::AbstractCoordinatePoint{T}) where {T} = typeof(PT)(zero(T),zero(T),zero(T))`

Reply:
Ah, right. I changed it.